.gitignore

```diff
@@ -32,6 +32,7 @@ packages/next-auth/providers
 # copied from @auth/core
 packages/frameworks-*/**/providers
 packages/*/*.js
+!packages/*/typedoc.config.js
```

Comment:
have to convert some files to .js so this will prevent the file from being deleted by `clean` scripts